cvat-canvas/README.md

```diff
@@ -76,6 +76,7 @@ All methods are sync.
  ```canvas_shape_drawing```
 - Tags has a class ```canvas_tag```
 - Canvas image has ID ```canvas_image```
+- Grid on the canvas has ID ```canvas_grid_pattern```
```

Comment:
Does it make sense to have a prefix for all such elements like "cvat:canvas_grid_pattern"?

Reply:
@nmanovic 
What is a goal for that? To reduce a chance of id collision?

Reply:
Yes.  It is the only purpose. It is a reason why C++ has namespaces. For example, canvas_image looks very general.

Reply:
@nmanovic 
Okay, let's do it.